src/components/dialog/dialog.component.js

```diff
@@ -7,76 +7,17 @@ import Icon from '../icon';
 import Modal from '../modal';
 import Heading from '../heading';
 import ElementResize from '../../utils/helpers/element-resize';
+import {
+  DialogStyle,
+  DialogTitleStyle,
+  DialogContentStyle,
+  DialogInnerContentStyle
+} from './dialog.style';
 import './dialog.scss';
 
 const DIALOG_OPEN_HTML_CLASS = 'carbon-dialog--open';
 
-/**
- * A Dialog widget.
- *
- * == How to use a Dialog in a component:
- *
- * In your file
- *
- *   import Dialog from 'carbon-react/lib/components/dialog';
- *
- * To render a Dialog:
- *
- *   <Dialog onCancel={ customEventHandler } />
- *
- * The component rendering the Dialog must pass down a prop of 'open' in order to open the dialog.
- *
- * You need to provide a custom cancel event handler to handle a close event.
- *
- * @class Dialog
- * @constructor
- */
 class Dialog extends Modal {
-  static propTypes = assign({}, Modal.propTypes, {
-    /**
-     * Allows developers to specify a specific height for the dialog.
```

Comment:
Is it worth keeping the comments on the propTypes? I think the Info addon in Storybook consumes them...

Reply:
Is the comment ok :)? Coz comment is on removed part of the code

Reply:
👍